gui/src/routes/home/_l/transactions.tsx

```diff
@@ -126,12 +126,16 @@ interface IconProps {
 }
 
 function Icon({ account, tx }: IconProps) {
+  const colorClass = tx.status === 0 
+    ? 'text-destructive dark:text-destructive'     // red for failed/reverted
+    : 'text-[#22c55e] dark:text-[#4ade80]';       // green for successful
```

Comment:
these comments are redundant.
and as per the rest of the codebase, the intended way to select between styles is to use `clsx`

Also, using `text-[#....]` is a bad pattern. if colors are missing, then those should be added to tailwind's styles on @ethui/ui as a global color